test/conformance/kuttl/generate/policy/cornercases/pol-clone-create-on-trigger-deletion/01-manifests.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pol-clone-create-on-trigger-deletion-ns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pol-clone-create-on-trigger-deletion-configmap
+  namespace: pol-clone-create-on-trigger-deletion-ns
+data:
+  foo: bar
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: pol-clone-create-on-trigger-deletion-policy
+  namespace: pol-clone-create-on-trigger-deletion-ns
+spec:
+  rules:
+  - name: default-deny
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+          operations:
+          - DELETE
+    generate:
+      apiVersion: networking.k8s.io/v1
+      kind: NetworkPolicy
+      name: pol-clone-create-on-trigger-deletion-netowrkpolicy
+      namespace: pol-clone-create-on-trigger-deletion-ns
+      synchronize: true
+      data:
+        spec:
+          podSelector: {}
+          policyTypes:
+          - Ingress
+          - Egress
```

Comment:
@realshuting i think this should be a clone..the resource is not being cloned..WDYT

Reply:
Crap! I will send a PR to fix this.